crypto/rand/rand_unix.c

```diff
@@ -232,10 +223,10 @@ static ssize_t sysctl_random(char *buf, size_t buflen)
      * up to 256 bytes.
      * Just return an error on older NetBSD versions.
      */
-#if   defined(__NetBSD__) && __NetBSD_Version__ < 400000000
+#  if   defined(__NetBSD__) && __NetBSD_Version__ < 400000000
```

Comment:
please do not reformat this file, that makes the review unnecessarily difficult for me.

Reply:
hmm, it helped me in sorting out the hairball of #ifdef, should we cleanup while we are here or not?